src/device_batch_gemm.cc

```diff
@@ -733,18 +768,18 @@ void blas::batch::gemm(
 // -----------------------------------------------------------------------------
 /// @ingroup gemm
 //  an API for group of fixed size (in fact, fixed params)
-void blas::batch::gemm(
+void gemm(
     blas::Layout                              layout,
     std::vector<blas::Op>              const &transA,
     std::vector<blas::Op>              const &transB,
     std::vector<int64_t>               const &m,
     std::vector<int64_t>               const &n,
     std::vector<int64_t>               const &k,
     std::vector<std::complex<float> >  const &alpha,
-    std::vector<std::complex<float>*>  const &Aarray,     std::vector<int64_t> const &ldda,
-    std::vector<std::complex<float>*>  const &Barray,     std::vector<int64_t> const &lddb,
+    std::vector<std::complex<float>*>  const &Aarray, std::vector<int64_t> const &ldda,
```

Comment:
No space for `complex` vector (see previous remark).

Reply:
There are about 444 instances of this, so I'll leave that for later.
```
blaspp> git grep -P '<std::complex<\w+>\*?>' | wc
    344    2112   38048
blaspp> git grep -P '<std::complex<\w+>\*? >' | wc
    100     524    8304
blaspp> git grep -P '< std::complex<\w+>\*? >' | wc
     78     624    7644
```